migrations/versions/0356_add_webautn_auth_type.py

```diff
@@ -0,0 +1,40 @@
+"""
+
+Revision ID: 0356_add_webautn_auth_type
+Revises: 0355_add_webauthn_table
+Create Date: 2021-05-13 12:42:45.190269
+
+"""
+from alembic import op
+
+revision = '0356_add_webautn_auth_type'
+down_revision = '0355_add_webauthn_table'
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.execute("INSERT INTO auth_type VALUES ('webauthn_auth')")
+
+    op.drop_constraint('ck_users_mobile_or_email_auth', 'users', type_=None, schema=None)
+    op.create_check_constraint(
```

Comment:
When an alter table command is issues any writes to the table will be blocked during the execution of the command. To mitigate this we can add a `NOT VALID` command. Then in another PR we can validate the check constraint.
This will reduce any risk of the data migration blocking any commits to the users table.